shared/actions/signup.tsx

```diff
@@ -50,6 +51,12 @@ const showErrorOrCleanupAfterSignup = (state: TypedState) =>
     ? SignupGen.createRestartSignup()
     : RouteTreeGen.createNavigateAppend({parentPath: [loginTab], path: ['signupError']})
 
+// If the email was set to be visible during signup, we need to set that with a separate RPC.
+const setEmailVisibilityAfterSignup = (state: TypedState) =>
+  noErrors(state) &&
+  state.signup.emailVisible &&
+  SettingsGen.createEditEmail({email: state.signup.email, toggleSearchable: true})
```

Comment:
I got a black bar here on signing up:
https://github.com/keybase/client/blob/8cac5949400d62f5b259b5d793c27068bd696ba9/shared/actions/settings.tsx#L414
`TypeError: Cannot read property 'get' of null`. Maybe because it's disabled on the server for the user I signed up? In any case, maybe it makes more sense to call the RPC here and not depend on the state being predictable during signup / login.

Reply:
I think I'll change that action to be `setVisibility` instead of `toggleVisibility`. Then we won't need to check `email.emails` at all.